File: Python_백준/10818_최소, 최대.py
# N개의 정수가 주어진다.
# 이때, 최솟값과 최댓값을 구하는 프로그램을 작성하시오.
#
# 예제입력1
# 5
# 20 10 35 30 7
# 출력1
# 7 35

n = int(input())
lst = input().split()

for i in range(n):
    lst[i] = int(lst[i])

answer = ""

answer += str(max(lst)) + " " + str(min(lst))

print(answer)

File: Java_백준/10818_최소, 최대.java
//10818
package onearray;
import java.util.Scanner;
import java.util.Arrays;

public class minmax {

	public static void main(String[] args) {
		int max = -1000000;
		int min = 1000000;
		Scanner sc = new Scanner(System.in);
		String length = sc.nextLine();
		String[] ary = sc.nextLine().split(" ");
		for(String w : ary) {
			if(Integer.parseInt(w) > max) {
				max = Integer.parseInt(w);
			}
			if(Integer.parseInt(w) < min) {
				min = Integer.parseInt(w);
			}
		}
		System.out.println(min + " " + max);
	}

}
